jinja2/ext.py

```diff
@@ -434,6 +442,59 @@ class AutoEscapeExtension(Extension):
     pass
 
 
+class DebugExtension(Extension):
+    """
+    A ``{% debug %}`` tag that dumps the available variables, filters and tests.
+    Typical usage like this:
+
+    .. codeblock:: html
+        <pre>{% debug %}</pre>
+
+    produces output like this:
+
+    ``
```

Comment:
Use `::` for Python code blocks. Double backticks are only for inline code.

Reply:
OK. Can you educate me on where to look to see what is produced? I've tried running sphinx by hand, and searched in vain for where this stuff ends up under the docs/_build tree. (I'm a sphinx newb).